### ISSUE TITLE
InvalidTemplate Error

### ISSUE THREAD

**Issue by frarin** [AgicCloud/Demo.Aks.Performance#19]
![image](https://user-images.githubusercontent.com/98345119/169494418-ca6f27ea-3e1f-4e30-8b13-b4f3bf1d3d00.png)

**Comment by macel94** [comment]
@frarin 
in your repository there is an error, that is an ENV variable name mismatch.
https://github.com/frarin/Demo.Aks.Performance/blob/main/.github/workflows/main.yml#L10
https://github.com/frarin/Demo.Aks.Performance/blob/main/.github/workflows/main.yml#L188

please delete your test env and retry, making sure that your code is correctly aligned with this repo.
thank you for your help

**Comment by frarin** [comment]
@MACEL94 @mosca97 my code is correctly aligned with this repo, I deleted all the resources in my "unique-resource-group-aks-demo" resource group and I tried to run again the manual-deploy, but I got the same error...

**Comment by macel94** [comment]
Please check the lines I previously commented. You have a wrong env variable name. Maybe during a merge git automatically resolved the conflict and you didn't notice it? But if you check the variable name you have in your repo with our one you will see the difference

**Comment by frarin** [comment]
C**AH**OSMESHEXPERIMENTNAME <--> C**HA**OSMESHEXPERIMENTNAME

Got it, let me try again, I'll keep you updated.

**Comment by frarin** [comment]
@MACEL94 
![image](https://user-images.githubusercontent.com/98345119/169584527-a11e0589-e70a-47a7-9e1c-6002027ee515.png)

**Comment by macel94** [comment]
@frarin in your last run, where i believe you found the error(please link me the run next time, it's difficult to understand what you mean from a screenshot) you checked out a version of your repo where the variable was still wrong.
proof: https://github.com/frarin/Demo.Aks.Performance/actions/runs/2357089631 --> this is a re-run that got this commit when checked out: https://github.com/frarin/Demo.Aks.Performance/commit/23aaa54ebc0f6cca2ae945039f4dbd15f47fddc9

and you can see at line 10: https://github.com/frarin/Demo.Aks.Performance/commit/23aaa54ebc0f6cca2ae945039f4dbd15f47fddc9#diff-7829468e86c1cc5d5133195b5cb48e1ff6c75e3e9203777f6b2e379d9e4882b3R10

you still had the wrong variable.
please try running the workflow with a new run(that will checkout your latest version of sources) instead of re-running the old one(that still refers to an older version of your repo).

thank you
